packages/space-opera/src/components/home/home.ts

```diff
@@ -0,0 +1,160 @@
+    header: 'Import/ Export',
+    body:
+        'Import or export your GLB models as well as insert or copy &lt;model-viewer&gt; HTML snippets.'
+  },
+  edit: {
+    icon:
+        'https://fonts.gstatic.com/s/i/googlematerialicons/create/v6/grey600-24dp/1x/gm_create_grey600_24dp.png',
+    header: 'Edit',
+    body: 'Adjust your lighting, hotspots, or posters in finer detail.'
+  },
+  camera: {
+    icon:
+        'https://fonts.gstatic.com/s/i/materialiconsextended/photo_camera/v6/grey600-24dp/1x/baseline_photo_camera_grey600_24dp.png',
+    header: 'Camera',
+    body:
+        'Update the camera settings including adding interactivity, rotation, and targets.'
+  },
+  materials: {
+    icon:
+        'https://fonts.gstatic.com/s/i/materialiconsextended/color_lens/v7/grey600-24dp/1x/baseline_color_lens_grey600_24dp.png',
+    header: 'Materials',
+    body:
+        'Modify properties of materials such as color, metallic roughness, normal maps, etc.'
+  },
+  inspector: {
+    icon:
+        'https://fonts.gstatic.com/s/i/materialiconsextended/search/v7/grey600-24dp/1x/baseline_search_grey600_24dp.png',
+    header: 'Inspector',
+    body: 'Visualize the gltf json string generated from your imported model.'
+  }
+};
+
+/**
+ * Home Container Card
+ */
+@customElement('home-container-card')
+export class HomeContainerCard extends ConnectedLitElement {
+  static styles = homeStyles;
+  @property({type: String, reflect: true}) icon = '';
+  @property({type: String}) homeCardHeader = '';
+  @property({type: String}) homeCardBody = '';
+
+  render() {
+    return html`
+      <div class="CardContainer">
+        <div class="CardContent">
+          <img src=${this.icon}>
+        </div>
+        <div class="CardContent text">
+          <div class="HomeCardHeader">${this.homeCardHeader}</div> 
+          <div class="HomeCardContent">${this.homeCardBody}</div>
+        </div>
+      </div>
+    `;
+  }
+}
+
+/**
+ * Home Container
+ */
+@customElement('home-container')
+export class HomeContainer extends ConnectedLitElement {
+  static styles = homeStyles;
+
+  render() {
+    return html`
+      <me-expandable-tab tabName="Home" .open=${true} .sticky=${true}>
+        <div slot="content">
+          <div class="note">
+            Welcome to the model viewer editor where you can generate &lt;model-viewer&gt; HTML snippets, as well as edit GLBs.
+          </div>
+          <a href="https://policies.google.com/privacy" style="color: black">Privacy</a>
+        </div>
+      </me-expandable-tab>
+      <me-expandable-tab tabName="Modules" .open=${true}>
+        <div slot="content">
+          <me-card title="File Manager">
+            <div slot="content">
+              <home-container-card 
+                icon=${CARD_CONTENT.save.icon}
+                homeCardHeader=${CARD_CONTENT.save.header}
+                homeCardBody=${CARD_CONTENT.save.body}
```

Comment:
Since this code is all repeated, is there any easy way to do something like `<home-container-card content=CARD_CONTENT.save>` instead?

Reply:
yep, good call, let me do that

Reply:
done